docs/pages/router/reference/async-routes.mdx

```diff
@@ -24,27 +25,61 @@ All Routes are wrapped inside a suspense boundary and are loaded asynchronously.
 
 Loading errors are handled in the parent route, via the [`ErrorBoundary`](/routing/error-handling/#errorboundary) export.
 
-Async routes cannot be statically filtered during development, so all files will be treated as routes even if they don't export a default component. After the component is bundled and loaded, then any invalid route will use a fallback warning screen. This behavior is development-only and will not be present in production.
+Async routes cannot be statically analyzed during development, so all files will be treated as routes even if they don't export a default component. After the component is bundled and loaded, any invalid route will use a fallback warning screen.
 
-## Setup
+For those familiar with advanced bundling techniques, the async routes feature is comprised of [React Suspense](https://react.dev/docs/concurrent-mode-suspense), [route-based bundle splitting](https://legacy.reactjs.org/docs/code-splitting.html#route-based-code-splitting) and [lazy bundling](https://github.com/react-native-community/discussions-and-proposals/blob/main/proposals/0605-lazy-bundling.md) (in development).
 
-Enable the feature by setting the `asyncRoutes` option to `development` in your [app config](/versions/latest/config/app/):
+## Setup
 
-```json app.json
-{
-  "expo": {
-    "plugins": [
-      [
-        "expo-router",
-        {
-          "origin": "https://acme.com",
-          "asyncRoutes": "development"
-        }
-      ]
-    ]
-  }
-}
-```
+Enable the feature by setting the `asyncRoutes` option in the Expo Router Config Plugin of your [app config](/versions/latest/config/app/):
+
+<Tabs>
+
+<Tab label="SDK 49">
+   
+    ```json app.json
+    {
+      "expo": {
+        "plugins": [
+          [
+            "expo-router",
+            {
+              "origin": "https://acme.com",
+              "asyncRoutes": "development"
+            }
+          ]
+        ]
+      }
+    }
+    ```
+
+</Tab>
+
+
+<Tab label="SDK 50 and above">
+
+    > SDK 50 supports production bundle splitting, set `asyncRoutes` to `true` to enable it.
```

Comment:
```suggestion
    > SDK 50 supports production bundle splitting. Set `asyncRoutes` to `true` to enable it.
```

Let's split the sentence here instead of adding a pause.